cmd/plan.go

```diff
@@ -146,8 +154,8 @@ func GetPlanCommand() *cobra.Command {
 	planCmd.Flags().StringVarP(&flags.srcpath, sourceFlag, "s", ".", "Specify source directory.")
 	planCmd.Flags().StringVarP(&flags.planfile, planFlag, "p", common.DefaultPlanFile, "Specify a file path to save plan to.")
 	planCmd.Flags().StringVarP(&flags.name, nameFlag, "n", common.DefaultProjectName, "Specify the project name.")
-	planCmd.Flags().StringVarP(&flags.customizationsPath, customizationsFlag, "c", common.DefaultCustomizationFolder, "Specify directory where customizations are stored.")
-	planCmd.Flags().StringSliceVarP(&flags.configs, configFlag, "f", []string{common.DefaultConfigFilePath}, "Specify config file locations.")
+	planCmd.Flags().StringVarP(&flags.customizationsPath, customizationsFlag, "c", "", "Specify directory where customizations are stored.")
+	planCmd.Flags().StringSliceVarP(&flags.configs, configFlag, "f", []string{}, "Specify config file locations.")
```

Comment:
Specify in the comments that if the default config file is found, it will be considered. If not ignored.

Reply:
done